Doc/c-api/init.rst

```diff
@@ -921,8 +921,8 @@ interpreter (created automatically by :c:func:`Py_Initialize`).  Python
 supports the creation of additional interpreters (using
 :c:func:`Py_NewInterpreter`), but switching between interpreters via the
 ``PyGILState_*`` API is unsupported. On the other hand, *creating*
-sub-interpreters (whether they have a per-interprter GIL or not in
-the sense of :pep:`684`) still require to hold the :term:`GIL`.
+sub-interpreters (whether they have a per-interpreter :term:`GIL` or not in
```

Comment:
I've looked at the rest of the paragraph and I don't think we linkify the GIL so you can simply keep `GIL` for both occurrences. I'm going to sleep (it's 2 AM here) but otherwise I think it's fine (if what I rephrased was indeed what you expected).

Reply:
Ok, will do.